### PR TITLE
feat: add RDOC to token list

### PR DESCRIPTION
**Pull request by jurajpiar**
## What

-   Hardcode the value of RDOC in the price feeder to always return 1.

## Why

-  Add support for RDOC as token.

## Refs

-   [ROLLUP-158](https://rsklabs.atlassian.net/browse/ROLLUP-158)

### CODE REVIEW

File: core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs [hunk 1]
```diff
@@ -119,3 +206,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
...
+    use std::env;
+    use zksync_types::{Address, Token, TokenId, TokenKind, TokenPrice};
+
+    #[tokio::test]
+    async fn should_return_one_for_rdoc() {
+        const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+        const RDOC_SYMBOL: &str = "RDOC";
+        const RDOC_VALUE: u32 = 1;
+
+        env::set_var("DATABASE_URL", DATABASE_URL);
```
Comment (antomor) [inline core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs:223]: are we sure that we need this? have we tried to run the tests with `zk` as they are supposed to be run?

File: core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs [hunk 2]
```diff
@@ -119,3 +206,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
...
+                Err(PriceError::token_not_found("Wrong token"))
+            }
+        }
+
+        let token = TokenLike::Symbol(String::from(RDOC_SYMBOL));
+
+        let connection_pool = ConnectionPool::new(Some(1));
+        let ticker_api = TickerApi::new(connection_pool, FakeTickerApi);
+
+        let actual_qoute = FeeTickerAPI::get_last_quote(&ticker_api, token)
```
Comment (antomor) [inline core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs:239]:
```suggestion
        let actual_quote = FeeTickerAPI::get_last_quote(&ticker_api, token)
```

File: core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs [hunk 3]
```diff
@@ -119,3 +206,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
...
+        assert_eq!(actual_qoute.usd_price.to_u32().unwrap(), RDOC_VALUE);
+    }
+
+    #[tokio::test]
+    async fn should_return_value_from_cache() {
+        const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+        const TEST_TOKEN_SYMBOL: &str = "TEST";
+        const TOKEN_VALUE: u32 = 5;
+
+        env::set_var("DATABASE_URL", DATABASE_URL);
```
Comment (antomor) [inline core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs:252]: Are we sure that we need this? Have we tried to run the tests with `zk` as they are supposed to be run?

File: core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs [hunk 4]
```diff
@@ -119,3 +206,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
...
+
+        let price_cache_map = HashMap::from([(
+            test_token.id,
+            TokenCacheEntry::new(test_token_price.clone(), Instant::now()),
+        )]);
+        let price_cache = Arc::new(Mutex::new(price_cache_map));
+
+        let ticker_api = TickerApi::with_price_cache(ticker_api, price_cache);
+
+        let actual_qoute = FeeTickerAPI::get_last_quote(&ticker_api, test_token_symbol)
```
Comment (antomor) [inline core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs:290]:
```suggestion
        let actual_quote = FeeTickerAPI::get_last_quote(&ticker_api, test_token_symbol)
```

**Review by antomor — CHANGES_REQUESTED** [review]
@jurajpiar @AndresQuijano I'd verify if we really need that env variable or if we don't by using the `zk` cli

**Comment by jurajpiar** [comment]
actually this doesn't seem to compile so I'm not sure why it was ready for review.

**Pull request closed without merging**